sympy/physics/mechanics/jointsmethod.py

```diff
@@ -152,7 +153,8 @@ def _generate_bodylist(self):
     def _generate_loadlist(self):
         load_list = []
         for body in self.bodies:
-            load_list.extend(body.loads)
+            if isinstance(body, Body):
+                load_list.extend(body.loads)
```

Comment:
Is this because only `Body` instances have a `loads` attribute?

Reply:
Yes